environments/nwb_benchmarks.yaml

```
@@ -9,15 +9,14 @@ dependencies:
       - asv == 0.6.1
       - numba>=0.58.1  # Pin to specific version for cuda import
       - psutil
-      - pynwb
       - dandi
       - fsspec
       - requests
       - aiohttp
       - remfile
       - pyshark
-      - lindi
-      - hdmf @ git+https://github.com/hdmf-dev/hdmf.git@9b3282aa4999d2922f003f1b79ec7458ea3ddc5e  # required until PyNWB propagates changes
-      #- hdmf @ git+https://github.com/hdmf-dev/hdmf.git@expose_aws_region  # required until region fix is released
+      - lindi>=0.3.5
```

```suggestion
      - lindi>=0.3.6
```

Required for latest fix